scripts/deblur

```diff
@@ -94,7 +94,12 @@ def trim(seqs_fp, output_fp, trim_len):
               show_default=True, help="Discard sequences with an abundance "
               "value smaller than min-size")
 def dereplicate(seqs_fp, output_fp, min_size):
-    """Step 2a: dereplicate FASTA sequences"""
+    """
+    Step 2a: dereplicate FASTA sequences.
```

Comment:
I think we should remove them from here and just put good online documentation...